config

```diff
@@ -24,6 +24,7 @@ fatfree-3.8
 fuelphp-1.9
 kumbia-1.2
 laravel-10.2
```

Comment:
There is a conflict here with `v10.3`

Reply:
Please, try to merge from older to newer PR.
At the bottom are the older PRs.

Reply:
Owh